pyccel/ast/headers.py

```diff
@@ -179,6 +179,31 @@ def create_definition(self):
         funcs = []
         dtypes = []
 
+        def build_argument(var_name, dc):
```

Comment:
Can you please add a comment line that explains what this function does? It will be very useful for future developers looking into this code.